lua/starfall/libs_sh/players.lua

```diff
@@ -824,6 +825,16 @@ if SERVER then
 			Ply_Kill(ent)
 		end
 	end
+	
+	--- Attempts to force the target into a vehicle.
+	--- Requires 'player.enterVehicle' permission on the player.
+	-- @server
+	-- @param Vehicle vehicle
+	function player_methods:enterVehicle(vehicle)
+		local ent = getply(self)
+		checkpermission(instance, ent, "player.enterVehicle")
+		ent:EnterVehicle(vhunwrap(vehicle))
```

Comment:
Can you add Ply_EnterVehicle to the list on line 95 (It's alphabetically sorted) and use that here?